src/PrestaShopBundle/Resources/config/services/bundle/controller.yml

```diff
@@ -2,29 +2,35 @@ services:
   _defaults:
     public: true
 
+  _instanceof:
+    # cannot use autocnfigure under _instanceof so we need to define it here
+    PrestaShopBundle\Controller\Admin\PrestaShopAdminController:
+      autowire: true
+      public: true
+      tags:
+        - 'controller.service_arguments'
+        - 'container.service_subscriber'
+    PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController:
```

Comment:
If we have this do we still need the `ContainerInjectionPass` ? These two solutions seem redundant no?

Reply:
If we go with this solution only we should comment here that this rule handles the fact that controllers based on `FrameworkBundleAdminController` are the only one where we still inject the full container

Reply:
See if the modif in the class itself is enough, if yes then this should be cleaned